Makefile

```diff
@@ -210,18 +208,10 @@ docker.$(PROXY): build.$(PROXY)
 	docker build -t $(DOCKER_REPO)/cellery-hub-proxy:$(VERSION) .
 
 .PHONY: docker.$(DOCKER_AUTH)
-docker.$(DOCKER_AUTH): build.$(DOCKER_AUTH)
-	rm -rf ./docker/$(DOCKER_AUTH)/target
-	rm -rf ./docker/$(DOCKER_AUTH)/$(AUTH_SERVER)/target
-	mkdir ./docker/$(DOCKER_AUTH)/target/
-	cp ./components/$(DOCKER_AUTH)/target/authentication ./docker/$(DOCKER_AUTH)/target/
-	cp ./components/$(DOCKER_AUTH)/target/authorization ./docker/$(DOCKER_AUTH)/target/
-	mkdir ./docker/$(DOCKER_AUTH)/$(AUTH_SERVER)/target/
-	cp ./components/$(DOCKER_AUTH)/target/auth_server ./docker/$(DOCKER_AUTH)/$(AUTH_SERVER)/target/
-	cd ./docker/$(DOCKER_AUTH); \
+docker.$(DOCKER_AUTH):
+	cd ./docker/$(AUTH_SERVER); \
+	docker build -t $(DOCKER_REPO)/$(AUTH_SERVER):latest .
```

Comment:
Don't hard-code the tag. Please use the VERSION variable.

Reply:
Used the commit hash as the image version. Since we need to keep this base image separate.